Dockerfile

```diff
@@ -1,16 +1,11 @@
-FROM golang:1.8.3-alpine
-
+FROM golang:1.8.3-alpine3.6 as unsee-builder
 COPY . /go/src/github.com/cloudflare/unsee
 
 ARG VERSION
+RUN apk add --update make git nodejs nodejs-npm
+RUN make -C /go/src/github.com/cloudflare/unsee VERSION="${VERSION:-dev}" unsee-static
 
-RUN apk add --update --no-cache --virtual .build-dependencies \
-        make git nodejs && \
-    make -C /go/src/github.com/cloudflare/unsee VERSION="${VERSION:-dev}" && \
-    mv /go/src/github.com/cloudflare/unsee/unsee /bin/unsee && \
-    rm -fr /go/src && \
-    apk del .build-dependencies
-
+FROM gcr.io/distroless/base
```

Comment:
I was struggling to find info as to what the distroless base image contains, but this covers it:

https://github.com/GoogleCloudPlatform/distroless/blob/master/base/README.md

We don't need glibc or libssl if the binary is statically compiled.

The code to use `scratch` would look roughly like:

```
# Build stage
FROM golang:1.8 as build
WORKDIR /go/src/github.com/foo/bar
RUN apt-get update
RUN apt-get upgrade -y ca-certificates
RUN apt-get install -y git make
COPY . /go/src/github.com/foo/bar
RUN make # or similar

# Main stage
FROM scratch
COPY --from=build /etc/ssl/certs /etc/ssl/certs
COPY --from=build /go/bin/example /
ENTRYPOINT ["/example"]
```

...which gives you an image containing only the statically-compiled binary and the CA certificates copied from Debian.

I'm not suggesting you change it, I think using distroless is fine.